src/includes/performance/configure-sample-rate/javascript.vue.mdx

```diff
@@ -41,15 +41,19 @@ Sentry.init({
 });
 ```
 
-### Tracking Child Components
+## Configure Tracing Insturmentation
 
-If you want to track child components, and see more details about the rendering process, configure the integration to track them all:
+If you want to track child components or use additional hooks to see more details about the rendering process, you can configure the SDK using options below:
```

Comment:
Can we add a note that this adds spans to the current active transaction on the scope?